postreise/analyze/generation/summarize.py

```diff
@@ -26,12 +26,12 @@ def sum_generation_by_type_zone(
     """
     _check_scenario_is_in_analyze_state(scenario)
 
-    pg = scenario.state.get_pg()
+    pg = scenario.get_pg().copy()
```

Comment:
Why we would like to do a `copy` here?

Reply:
Because the `change_time_zone` function operates in place. It results that after calling `sum_generation_by_type_zone` with `time_zone` set, `scenario.get_pg()` will return a data frame with timestamps in the time zone set.

Reply:
Ah, I see. I thought it made copies implicitly by returning `pg`. But we modified it in-place before that.